rock_paper_scissors/rps.py

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/python
 
 import sys
+from itertools import product
 
 def rock_paper_scissors(n):
-  pass 
-
+  print(n)
+  array = ['rock', 'paper', 'scissors']
+  if n == 1:
+    return [['rock'], ['paper'], ['scissors']]
+  else:
+    # print(f'This is the list === {list(product(array, repeat = n))}') #returns a tuple
+    combinations = [list(x) for x in list(product(array, repeat = n))]
```

Comment:
good list comprehension here. itertools really does this whole thing for you :smile: